frontend/packages/patternfly/src/components/notification-drawer/notification-drawer.tsx

```diff
@@ -0,0 +1,94 @@
+import * as _ from 'lodash';
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+import {
+  EmptyState,
+  EmptyStateBody,
+  EmptyStateSecondaryActions,
+  EmptyStateVariant,
+  Title,
+} from '@patternfly/react-core';
+import {
+  Drawer,
+  DrawerContent,
+  DrawerPanelContent,
+} from '@patternfly/react-core/dist/js/experimental/components/Drawer';
+
+import NotificationDrawerHeading from './notification-drawer-heading';
+import NotificationTypeHeading from './notification-type-heading';
+import { NotificationEntryProps } from './notification-entry';
+
+export enum NotificationTypes {
+  info = 'info',
+  warning = 'warning',
+  critical = 'danger',
+  success = 'success',
+  update = 'update',
+}
+
+const emptyState = (toggleDrawer) => (
+  <EmptyState variant={EmptyStateVariant.full} className="co-status-card__alerts-msg">
+    <Title headingLevel="h5" size="lg">
+      No critical alerts
+    </Title>
+    <EmptyStateBody>
+      There are currently no critical alerts firing. There may be firing alerts of other severities
+      or silenced critical alerts however.
+    </EmptyStateBody>
+    <EmptyStateSecondaryActions>
+      <Link to="/monitoring/alerts" onClick={toggleDrawer}>
+        View all alerts
+      </Link>
+    </EmptyStateSecondaryActions>
+  </EmptyState>
+);
+
+const NotificationDrawer: React.FC<NotificationDrawerProps> = ({
+  isDrawerExpanded,
```

Comment:
Curious what the PF pattern is, and this should probably align with it, But I always tend to think:

```
NotificationDrawer
- expanded
- toggle
```
vs
```
NotificationDrawer
- isDrawerExpanded
- toggleNotificationDrawer
```
The context of `NotificationDrawer` is provided in the component name, so the prop names can be pretty short.

Reply:
@benjaminapetersen there will be some changes when this is exported to patternfly. This is a good catch. Will update these to remove the component name as noted.

Reply:
@benjaminapetersen updated to remove component name thanks!